pkg/yurthub/transport/transport.go

```diff
@@ -134,6 +134,7 @@ func (tm *transportManager) start() {
 			tm.closeAll()
 			lastCert = curr
 		} else if lastCert != nil && curr != nil {
+			//nolint:staticcheck
 			if lastCert == curr {
 				// cert is not rotate, just wait
 			} else {
```

Comment:
Maybe we can avoid using the `//nolint:staticcheck` comment by using klog to output log in the empty if/else branch(we can set this log level to 5)?

Reply:
agree